consensus/types/src/light_client_update.rs

```diff
@@ -259,6 +248,150 @@ impl<E: EthSpec> LightClientUpdate<E> {
 
         Ok(update)
     }
+
+    // Implements spec prioritization rules:
+    // Full nodes SHOULD provide the best derivable LightClientUpdate for each sync committee period
+    // ref: https://github.com/ethereum/consensus-specs/blob/113c58f9bf9c08867f6f5f633c4d98e0364d612a/specs/altair/light-client/full-node.md#create_light_client_update
+    pub fn is_better_light_client_update(
+        &self,
+        new: &LightClientUpdate<E>,
+        chain_spec: &ChainSpec,
+    ) -> Result<bool, Error> {
+        // Compare super majority (> 2/3) sync committee participation
+        let max_active_participants = new.sync_aggregate().sync_committee_bits.len();
+
+        let new_active_participants = new.sync_aggregate().sync_committee_bits.num_set_bits();
+        let prev_active_participants = self.sync_aggregate().sync_committee_bits.num_set_bits();
+
+        let new_has_super_majority =
+            new_active_participants.safe_mul(3)? >= max_active_participants.safe_mul(2)?;
+        let prev_has_super_majority =
+            prev_active_participants.safe_mul(3)? >= max_active_participants.safe_mul(2)?;
+
+        if new_has_super_majority != prev_has_super_majority {
+            return Ok(new_has_super_majority & !prev_has_super_majority);
```

Comment:
Are you using Bitwise And intentionally here? Took me a while to verify this statement. The spec uses `new > old`, which is actually equivalent to your statement. However since we only consider `new != old` the statement can be simplified just to `new`.

| new | old | new > old | new & !old | !old |
| --- | --- | --------- | ---------- | ---- |
| 1   | 0   | 1         | 1          | 1    |
| 0   | 1   | 0         | 0          | 0    |

```suggestion
            return Ok(new_has_super_majority);
```

Reply:
sure i'll simplify this. the bitwise and was actually forced on me by clippy!